### ISSUE TITLE
Border width for window with custom CSS not working with Hyper 1.4.x

### ISSUE THREAD

**Issue by lidin** [zeit/hyper#2138]
Trying to set the border-width property of the css option member, but it doesn't update the window border. My configuration file is below, which works on prior versions (<1.3.3):

```
module.exports = {
  config: {
    // default font size in pixels for all tabs
    fontSize: 12,

    // `BEAM` for |, `UNDERLINE` for _, `BLOCK` for █
    cursorShape: 'BLOCK',

    // custom css to embed in the main window
    css:
      `.hyper_main {
        border-width: 0px;
      }`,

    // custom css to embed in the terminal window
    termCSS: '',

    // the full list. if you're going to provide the full color palette,
    // including the 6 x 6 color cubes and the grayscale map, just provide
    // an array here instead of a color map object
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
    },

    // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
    // if left empty, your system's login shell will be used by default
    // make sure to use a full path if the binary name doesn't work
    // (e.g `C:\\Windows\\System32\\bash.exe` instead of just `bash.exe`)
    // if you're using powershell, make sure to remove the `--login` below
    shell: '/usr/local/bin/zsh',

    // for setting shell arguments (i.e. for using interactive shellArgs: ['-i'])
    // by default ['--login'] will be used
    shellArgs: ['--login'],

    // for environment variables
    env: {},

    // set to false for no bell
    bell: false,

    // if true, selected text will automatically be copied to the clipboard
    copyOnSelect: false,

    // for advanced config flags please refer to https://hyper.is/#cfg

    // hyper-tabs-enhanced plugin
    hyperTabs: {
      trafficButtons: true
    },

    // hyper-statusline plugin
    hyperStatusLine: {
      footerTransparent: false
    },

    // hyperterm-alternatescroll plugin
    alternateScroll: {
      // 1 to 100 is supported
      scrollSpeed: 100
    },

    // hyper-pane plugin
    paneNavigation: {
      indicatorPrefix: '',
      indicatorStyle: {
        position: 'absolute',
        top: '0px',
        left: '14px',
        fontSize: '10px'
      },
    },
  },

  /* 'hyper-cat', */
  plugins: [
    "hyper-pane",
    "hyperterm-close-on-left",
    "hyper-hide-title",
    'hyperterm-alternatescroll',
    'hypersolar-dark',
    'hyper-blink',
    // 'hyperatompunk',
    // 'hyper-statusline',
    'hyperterm-bold-tab',
  ],

  // in development, you can create a directory under
  // `~/.hyper_plugins/local/` and include it here
  // to load it and avoid it being `npm install`ed
  localPlugins: []
};
```
        -
        Hyper 1.4.1
        Electron 1.6.12
        darwin x64 16.7.0

**Comment by lidin** [comment]
Never mind, realized now that the property have been renamed to '_main'.

**Comment by chabou** [comment]
this is a regression.
Fixed by #2139.
